aibot.py

```diff
@@ -330,9 +341,23 @@ def handle_conversation(say, payload, is_dm=False):
             content=response,
         )
     else:
-        response = get_text(prompt_messages)
-        response = response.rsplit('[name_separator]', 1)[-1]
-        say(response, response_type="in_channel")
+        # this should stream the results to the user by posting and editing a message ...
```

Comment:
Wicked cool!

Do you think Abby could revert to her prior behavior, if a person asked politely using a particular syntax?